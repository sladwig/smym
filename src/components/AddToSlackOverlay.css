.add-to-slack-overlay {
    left: 50%;
    margin: 0 62px;
    padding: 64px;
    border-radius: 32px;
    display: flex;
    flex-direction: column;
    justify-content: center;
    align-items: center;
    background: var(--whiteish);
}
.welcome {
    margin-bottom: 32px;
    margin-top: 32px;
}