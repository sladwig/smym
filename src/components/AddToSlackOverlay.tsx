import React, { useState } from 'react';
import './AddToSlackOverlay.css';

export const AddToSlackOverlay = ({}) => {
    const [show, set] = useState(false);
    return (
        <div className="add-to-slack-overlay">
            <div className="welcome heading-1">welcome</div>

            <div onClick={() => set(true)}>
                <img
                    alt="Add to Slack"
                    height="40"
                    width="139"
                    src="https://platform.slack-edge.com/img/add_to_slack.png"
                    srcSet="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x"
                />
            </div>
            <div>&nbsp;{show && 'not working yet'}</div>
        </div>
    );
};
