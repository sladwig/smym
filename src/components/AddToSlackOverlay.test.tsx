import { render } from '@testing-library/react';
import React from 'react';
import { AddToSlackOverlay } from './AddToSlackOverlay';

describe('AddToSlackOverlay', () => {
    test('renders', () => {
        const { container } = render(<AddToSlackOverlay />);
        expect(container).toMatchSnapshot();
    });
});
