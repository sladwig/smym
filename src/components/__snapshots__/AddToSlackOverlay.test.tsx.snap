// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AddToSlackOverlay renders 1`] = `
<div>
  <div
    class="add-to-slack-overlay"
  >
    <div
      class="welcome heading-1"
    >
      welcome
    </div>
    <div>
      <img
        alt="Add to Slack"
        height="40"
        src="https://platform.slack-edge.com/img/add_to_slack.png"
        srcset="https://platform.slack-edge.com/img/add_to_slack.png 1x, https://platform.slack-edge.com/img/add_to_slack@2x.png 2x"
        width="139"
      />
    </div>
    <div>
      Â 
    </div>
  </div>
</div>
`;
